.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,29 @@
 # What
+
 <!--- Describe your changes. -->
 
 # Why
+
 <!--- What problem does this change solve? -->
 <!--- Provide a link if you are addressing an open issue. -->
 
 # How Tested
+
 <!--- How did you test your change. What tests have you added. -->
 
 # Process & policy checklist
+
 <!--- Review the list and check the boxes that apply. -->
 
 - [ ] I have updated the project documentation to reflect my changes if necessary.
 - [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) documentation.
 
+<!--- If the PR has ARM Template changes check the boxes that apply. -->
```

Comment:
Would a header be better here? Like for **Is this a breaking change?**

**Does this PR contain changes to any of the ARM Templates?**